Sources/StreamChat/ChatClient_Tests.swift

```diff
@@ -893,43 +893,6 @@ class ChatClient_Tests: StressTestCase {
         XCTAssertEqual(testEnv.clientUpdater?.disconnect_called, true)
     }
     
-    func test_disconnectCalled_whenBackgroundTaskFinishes() {
-        // Create a new chat client
-        let client = ChatClient(
-            config: inMemoryStorageConfig,
-            workerBuilders: workerBuilders,
-            eventWorkerBuilders: [],
-            environment: testEnv.environment
-        )
-        client.connectAnonymousUser()
-        testEnv.clientUpdater?.reloadUserIfNeeded_completion?(nil)
-        
-        // Simulate access to `webSocketClient` so it is initialized
-        _ = client.webSocketClient
-        
-        // Simulate .connected state to obtain connection id
-        let connectionId: ConnectionId = .unique
-        testEnv.webSocketClient?.connectionStateDelegate?
-            .webSocketClient(testEnv.webSocketClient!, didUpdateConnectionState: .connected(connectionId: connectionId))
-        
-        XCTAssertEqual(client.connectionId, connectionId)
-        
-        // Simulate going into background
-        testEnv.backgroundTaskScheduler?.startListeningForAppStateUpdates_onBackground?()
-        
-        // Assert that background task is created
-        XCTAssertEqual(testEnv.backgroundTaskScheduler?.beginBackgroundTask_called, true)
-        
-        // Simulate background task execution finish
-        testEnv.backgroundTaskScheduler?.beginBackgroundTask_expirationHandler?()
-        
-        // Assert that `disconnect` is called
-        XCTAssertEqual(testEnv.clientUpdater?.disconnect_called, true)
-        
-        // Assert that background task is cancelled
-        XCTAssertEqual(testEnv.backgroundTaskScheduler?.endBackgroundTask_called, true)
```

Comment:
hm, shouldn't we only get rid of last 2 lines (where we check that background task is cancelled)? 🤔

Reply:
Yes, you're right, it was accidental